extensions/DPLforum/DPLforum_body.php

```diff
@@ -90,7 +90,7 @@ function get( $name, $value = null, $parser = null ) {
 		return $value;
 	}
 
-	function link( &$parser, $count, $page = '', $text = '' ) {
+	function link( $parser, $count, $page = '', $text = '' ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)